### PR TITLE
Added dotnet80 stack

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
### What does this PR do?:
Added `dotnet80` stack

### Which issue(s) this PR fixes:
https://github.com/devfile/api/issues/1540

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

File: stacks/dotnet80/devfile.yaml
Comment (Jdubrick) [inline stacks/dotnet80/devfile.yaml:12]:
```suggestion
  version: 1.0.0
```

Since this is an entirely new stack I am inclined to say we should start the versioning to reflect that.

**Review by Jdubrick — CHANGES_REQUESTED** [review]

**Comment by Jdubrick** [comment]
Who is going to be the stack owner of this new stack?

**Comment by msivasubramaniaan** [comment]
> Who is going to be the stack owner of this new stack?

I am planning to own this stack and added the entry on https://github.com/devfile/registry/blob/main/.github/CODEOWNERS

File: .github/CODEOWNERS
Comment (thepetk) [inline .github/CODEOWNERS:17]: @msivasubramaniaan I think your handle is different one right?

**Review by thepetk — CHANGES_REQUESTED** [review]
LGTM in general, we need to fix the CODEOWNERS file before merging (both fixing the handle and giving sufficient access to the codeowner).

File: .github/CODEOWNERS
Comment (msivasubramaniaan) [reply .github/CODEOWNERS:17]: Yes I am from IDE Raptors. Shall I update with `/stacks/dotnet80/  @msivasub @IDERaptors`?

Comment (thepetk) [reply .github/CODEOWNERS:17]: No, as mentioned at the top of the file, the `che-team` is a required reviewer for all stack related PRs so we should keep them there. My comment is for your own handle I think the current one is not the same as yours right? So suggested change is:

```suggestion
/stacks/dotnet80/       @msivasubramaniaan @devfile/che-team
```

**Comment by thepetk** [comment]
Needs review from the @devfile/che-team too.

File: stacks/dotnet80/devfile.yaml
Comment (svor) [inline stacks/dotnet80/devfile.yaml:1]:
```suggestion
schemaVersion: 2.2.2
```

**Review by svor — APPROVED** [review]
Just one suggestion to use 2.2.2 schema version
screenshot-nimbusweb me-2024 07 23-18_12_48

**Review by svor — APPROVED** [review]
lgtm

**Review by Jdubrick — APPROVED** [review]

**Pull request merged**
